src/Templates/parts/nav.twig

```diff
@@ -10,6 +10,7 @@
       </button>
       {% endif %}
       <a class="navbar-brand" href="{{baseurl}}">MyRadio</a>
+      <a class="navbar-brand" href="{{homeurl}}">ury.org.uk</a>
```

Comment:
@Brookke you've already got `$short_name` and `$long_name` in the config you could use (probably shortname). Maybe '`short_name` Website' or something.